### DISCUSSION TITLE
Force Light / Dark Mode

### DISCUSSION THREAD

**Discussion by TimRoenicke** [laravel/nova-issues#3979] [Q&A] [answered]
Is there an option to remove the picker and force one or the other?

**Comment by mdeimert** [comment]
Would be great to have this feature. 

Even if we hide the Switcher we should be able to force mode instead of using the system one.

**Comment by crynobone** [comment, marked as answer]
```js
// resources/js/nova.js

// force light mode
localStorage.novaTheme = 'light'
document.documentElement.classList.remove('dark')

// force dark mode
localStorage.novaTheme = 'dark'
document.documentElement.classList.add('dark')
```

Then add the following to `app\Providers\NovaServiceProvider.php`

```php
/**
 * Bootstrap any application services.
 *
 * @return void
 */
public function boot()
{
    parent::boot();

    Nova::withoutThemeSwitcher();
    Nova::script('nova-custom', resource_path('js/nova.js'));
}
```

**Comment by mdeimert** [comment]
Thank you for your suggestion @crynobone   but i think its still just a workaround - it should be added to the nova options or as a parameter to Nova::withoutThemeSwitcher();
